docs/usage/getting-started.rst

```diff
@@ -36,8 +36,15 @@ retrieve the correct value from the state dictionary:
 
 ::
 
-   from holodeck.sensors import Sensors
-   sensor_data = state[Sensors.LOCATION_SENSOR]
+   from holodeck import sensors
+   location_data = state[sensors.LocationSensor.sensor_type]
+
+Also, you can just provide the class name of the sensor as a string:
```

Comment:
This isn't quite right, you provide the name of the sensor, as defined in the scenario config https://holodeck.readthedocs.io/en/latest/packages/docs/scenarios.html#sensor-objects

If you don't provide a name for a sensor, it defaults to the class name of the sensor. So, for the purposes of this example, the example you added below is fine.